.github/actions/vtk-install-dep/action.yml

```diff
@@ -97,7 +103,9 @@ runs:
         -DBUILD_SHARED_LIBS=ON
         -DCMAKE_BUILD_TYPE=Release
         -DCMAKE_INSTALL_PREFIX:PATH=../vtk_install
+        -DCMAKE_OSX_ARCHITECTURES=${{ inputs.cpu }}
         -DCMAKE_PREFIX_PATH:PATH=$(pwd)/../install/
+        -DOpenVDB_CMAKE_PATH=$(pwd)/../install/lib/cmake/OpenVDB/
```

Comment:
I'm using OpenVDB_CMAKE_PATH becaause it seems be prefered with VTK and *may* work better with macOS in the future. Im not understanding it the issue fully yet, but I see no harm on using the canonical way.